package.json

```diff
@@ -43,6 +43,7 @@
     "react": "^16.3.0"
   },
   "dependencies": {
-    "prop-types": "^15.5.10"
+    "prop-types": "^15.5.10",
+    "ua-parser-js": "^0.7.19"
```

Comment:
The overhead of ua-parser-js is really big. Maybe we can just hard-code the regexes for IE and Edge?

Reply:
Updated the PR.